docs/component-usage-example.md

```
@@ -0,0 +1,500 @@
+          onChange={(e) => setAmount(e.target.value)}
+        />
+      </label>
+      <button type="submit">Contribute</button>
+    </form>
+  );
+};
+
+export default ContributionForm;
+```
+
+### Usage 
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)